transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketChannelTest.java

```diff
@@ -138,22 +138,26 @@ public void testExceptionHandlingDoesNotInfiniteLoop() throws InterruptedExcepti
     public void testAutoReadOffDuringReadOnlyReadsOneTime() throws InterruptedException {
         EventLoopGroup group = new EpollEventLoopGroup();
         try {
-            runAutoReadTest(group, EpollServerSocketChannel.class, EpollSocketChannel.class,
-                    new InetSocketAddress(0));
-            runAutoReadTest(group, EpollServerDomainSocketChannel.class, EpollDomainSocketChannel.class,
-                    EpollSocketTestPermutation.newSocketAddress());
+            for (boolean readOutsideEventLoopThread : new boolean[] {true, false}) {
+                runAutoReadTest(readOutsideEventLoopThread, group, EpollServerSocketChannel.class,
+                        EpollSocketChannel.class, new InetSocketAddress(0));
+                runAutoReadTest(readOutsideEventLoopThread, group, EpollServerDomainSocketChannel.class,
```

Comment:
could you not use a for loop put just have to test methods ? This will make it easier to see which one failed.